web/src/compose_validate.js

```diff
@@ -750,35 +750,26 @@ export function check_overflow_text() {
     // expensive.
     const text = compose_state.message_content();
     const max_length = realm.max_message_length;
+    const remainingCharacters = max_length - text.length;
     const $indicator = $("#compose-limit-indicator");
 
     if (text.length > max_length) {
         $indicator.addClass("over_limit");
         $("textarea#compose-textarea").addClass("over_limit");
         $indicator.html(
             render_compose_limit_indicator({
-                text_length: text.length,
+                remaining_length: remainingCharacters,
                 max_length,
```

Comment:
Previously, `max_length` was used in `compose_limit_indicator`. Since it has been removed, there is no need to pass the variable.

Reply:
Thank you for pointing that out. I've removed the unused max_length variable.